src/App.jsx

```diff
@@ -1,20 +1,25 @@
 // LIBRARY IMPORTS
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import {
 	BrowserRouter as Router,
 	Routes,
 	Route,
 } from 'react-router-dom';
 
 // LOCAL IMPORTS
-
+import { bioData } from './assets/bioData.js';
 import { Home, Layout, ProfessionalsPage } from './views';
 
 function App() {
+  // SET STATES
+  const [data, setData] = useState(bioData);
+
+  // SET REFERENCES
+  const fullDataSetRef = useRef(bioData);
```

Comment:
The useRef hook provides a way to keep a mutable reference to something without re-rendering the component when that something changes.